css-three-column-layout/styles.css

```diff
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+img {
+  width: 50%;
+  height: 50%;
+  justify-content: center;
+  align-items: center;
+  margin-top: 25%;
+  margin-left: 25%;
+}
+
+.circle-border {
+  display: flex;
+  border: solid;
+  border-color: white;
+  border-radius: 50%;
+  width: 100px;
+  height: 100px;
+  background-color: white;
+  margin-left: 30%;
+}
+
+body {
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  src: url("https://fonts.gstatic.com/s/lato/v15/S6uyw4BMUTPHjx4wXiWtFCc.woff2") format("woff2");
+  unicode-range:
+    u+0000-00ff,
+    u+0131,
+    u+0152-0153,
+    u+02bb-02bc,
+    u+02c6,
+    u+02da,
+    u+02dc,
+    u+2000-206f,
+    u+2074,
+    u+20ac,
+    u+2122,
+    u+2191,
+    u+2193,
+    u+2212,
+    u+2215,
+    u+feff,
+    u+fffd;
+}
+
+.column-full {
+  flex-basis: 100%;
+  text-align: center;
+  height: 100px;
+  padding-top: 50px;
+  background-color: rgb(0 149 255);
+  font-size: 1.4rem;
+  color: white;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  background-color: rgb(0 149 255);
+}
+
+.column-one-third {
+  align-items: center;
+  text-align: center;
+  padding-bottom: 50px;
+  flex-basis: calc(100% / 3);
+}
+
+h2 {
+  font-size: 0.9rem;
+  color: white;
+  font-weight: 550;
+}
+
+.column-one-third-row {
+  align-items: center;
+  text-align: center;
+  font-size: 0.6rem;
+  color: white;
+  margin-left: 40px;
+  margin-right: 40px;
+}
```

Comment:
I don't know the purpose of this, but no one element should be both a row and a column, and you shouldn't have variant classes for the `row` class, so you probably